docs/loadflow/loadflow.md

```diff
@@ -1,4 +1,33 @@
 ---
 title: Loadflow
-layout: todo
+layout: default
 ---
+
+The `com.powsybl.loadflow.LoadFlow` is the main entry point to run loadflow
+computations. It provides the static methods `run` and `runAsync`. These methods
+will run the computation and only differ in the way they return the results.
+The `run` method returns the results directly and can be used in most cases.
+The `runAsync` returns a `CompletableFuture` and can be used when a
+non-blocking computation is prefered. The `LoadFlow` class doesn't implement
+the computation directly, but instead relies on a
+`com.powsybl.loadflow.LoadFlowProvider` to implement it. This allows to use
+different loadflow algorithm implementations with the same code. Both the `run`
+and `runAsync` use the globally configured loadflow provider. `LoadFlow` also
+has a `find` method that allows to use a specific loadflow provider for this
+computation.
+
+**Note**: Powsybl provides a `com.powsybl.loadflow.mock.LoadFlowMock` that
+does nothing. An external implementation is available: [hades2](http://rte-france.github.io/hades2/index.html).
+
+# Configuration
+In order to use a loadflow provider, it must be accessible to Java's
+ServiceLoader. In most cases, this means adding a dependency to the loadflow
+provider artifact in the pom.xml file.
+
+To set the default loadflow provider, set the `default` property of the
+`load-flow` module configuration. When using the classic configuration, this
+means adding the following to your `config.yml`:
+```
+load-flow:
+  default: MyLoadFlowProvider
+```
```

Comment:
You should add an information about the fact that no configuration is required if there is only a single implementation in the class path?

Reply:
I Removed this short configuration paragraph and instead linked to the future configuration page from https://github.com/powsybl/powsybl.github.io/pull/75